bitsail-connectors/bitsail-connectors-legacy/bitsail-connector-hadoop/src/test/resources/hadoop_to_print.json

```diff
@@ -4,7 +4,8 @@
       "job_id": 313,
       "instance_id": 3123,
       "job_name": "bitsail_hadoop_to_print_test",
-      "user_name": "root"
+      "user_name": "root",
+      "metrics_reporter_type": "prometheus_pushgateway"
```

Comment:
It is recommended not to expose it to users, and related configurations can be placed in bitsail conf

Reply:
get it.

Reply:
the user need to define in the config if they want to use prometheus, and config the server port. But this test should be independent. We don't need to reply on hadoop test to verify metric's functionality. Please write UT for metric.